docs/modules/integrations/partials/authentication.adoc

```diff
@@ -1,36 +1,9 @@
 == Authentication
 
-* if the authentication through basic schema is available on your JIRA instance use `jira.username` and `jira.password` properties, both properties must be set
-* in case of two-factor authentication or any other complex authentication process it's possible to set session cookies into request headers like the following example shows (`jira.username` and `jira.password` properties must be empty in this case)
+* if the authentication through basic schema is available on your JIRA instance use `http.auth.username` and `http.auth.password` properties, both properties must be set
```

Comment:
why generic http properties are recommended but not Jira specific?

Reply:
since it's 0.4.0 didn't we decide to replace jira properties with http client ones?